src/layman/upgrade/upgrade_v2_0.py

```diff
@@ -172,3 +158,103 @@ def delete_layers_without_wfs_wms_available():
         assert not publ_info, f"{publ_info}"
 
         logger.info(f'    Deleting layer {workspace}.{layername} DONE!')
+
+
+def migrate_layers():
```

Comment:
I would rather use one cycle for all `1.23.0´x->2.0.0` migrations to make sure every publication is taken care of.